Tribler/Test/test_as_server.py

```diff
@@ -149,6 +153,16 @@ def tearDown(self, annotate=True):
             self.watchdog.print_all_stacks()
             raise RuntimeError("Couldn't stop the WatchDog")
 
+        if self.file_server:
+            deferred = self.file_server.stopListening()
```

Comment:
Don't forget that the `stoplistening()` may return None if there wasn't a connection, so wrap it in a `maybeDeferred`.